src/storageManager.ts

```diff
@@ -143,7 +143,7 @@ export function newStorageManager({moduleName, moduleType, advertiseKeys = true}
   const cookiesAreEnabled = function (done) {
     let cb = function (result) {
       if (result && result.valid) {
-        return checkCookieSupport();
+        return checkCookieSupport() && canSetCookie();
```

Comment:
should we always check due to  https://github.com/prebid/Prebid.js/security/code-scanning/257 ?